src/pages/SubscriptionService.tsx

```diff
@@ -185,6 +243,258 @@ class SubscriptionService extends Component<Props, SubscriptionServiceState> {
                 </div>
               )}
 
+              {storage.isApiAdmin && (
+                <div>
+                  <label className="m-0">{t("description")}</label>
+                  <input
+                    name="description"
+                    type="text"
+                    defaultValue={
+                      service.service_metadata
+                        ? service.service_metadata.description
+                        : undefined
+                    }
+                    onChange={this.handleInputChange}
+                    className="mb-4"
+                  />
+                </div>
+              )}
+
+              {storage.isApiAdmin && (
+                <div>
+                  <label className="m-0">{t("web_url")}</label>
+                  <input
+                    name="web_url"
+                    type="text"
+                    defaultValue={
+                      service.service_metadata
+                        ? service.service_metadata.web_url
+                        : undefined
+                    }
+                    onChange={this.handleInputChange}
+                    className="mb-4"
+                  />
+                </div>
```

Comment:
probably it's better to write a component and map the array of service metadata keys

Reply:
like this https://github.com/pagopa/io-developer-portal-frontend/blob/175160851-metadati/src/components/input/MetadataInput.tsx?

Reply:
avoid classes for components if you only need to use `render()`